.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu:
+    name: Ubuntu with Java 11
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout master branch
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Cache Maven packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+    - name: Install pre-requisites
+      run: ./dev/before_install
+    - name: Before build actions
+      run: ./dev/before
+    - name: Build
+      run: ./dev/main
+  macos:
+    name: macOS with Java 11
+    runs-on: macos-latest
+    steps:
+    - name: Checkout master branch
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Cache Maven packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+    - name: Install pre-requisites
+      run: ./dev/before_install
+    - name: Before build actions
+      run: ./dev/before
+    - name: Build
+      run: ./dev/main
+  windows:
+    name: Windows with Java 11
+    runs-on: windows-latest
+    steps:
+    - name: Checkout master branch
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Cache Maven packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+    - name: Install Universal ctags
+      run: choco install universal-ctags
+    - name: Build
+      run: mvn -B -V verify
```

Comment:
```suggestion
      run: ./mvnw -B -V verify
```
Maybe use `mvnw`?

Reply:
I tried that, does not work as the default shell is Powershell. Maven is installed by default the build runs fine.